src/Compilers/CSharp/Test/Emit2/Emit/EditAndContinue/EditAndContinueStateMachineTests.cs

```diff
@@ -2246,290 +2247,279 @@ .locals init (int V_0,
-    IL_00bd:  nop
-    IL_00be:  leave.s    IL_011a
-
-    IL_00c0:  ldarg.0
-    IL_00c1:  ldfld      ""System.Runtime.CompilerServices.TaskAwaiter C.<F>d__4.<>u__1""
-    IL_00c6:  stloc.3
-    IL_00c7:  ldarg.0
-    IL_00c8:  ldflda     ""System.Runtime.CompilerServices.TaskAwaiter C.<F>d__4.<>u__1""
-    IL_00cd:  initobj    ""System.Runtime.CompilerServices.TaskAwaiter""
-    IL_00d3:  ldarg.0
-    IL_00d4:  ldc.i4.m1
-    IL_00d5:  dup
-    IL_00d6:  stloc.0
-    IL_00d7:  stfld      ""int C.<F>d__4.<>1__state""
-
-    IL_00dc:  ldloca.s   V_3
-    IL_00de:  call       ""void System.Runtime.CompilerServices.TaskAwaiter.GetResult()""
-    IL_00e3:  nop
-    IL_00e4:  call       ""void C.End()""
-    IL_00e9:  nop
-    IL_00ea:  leave.s    IL_0106
-  }
-  catch System.Exception
-  {
-    IL_00ec:  stloc.s    V_4
-    IL_00ee:  ldarg.0
-    IL_00ef:  ldc.i4.s   -2
-    IL_00f1:  stfld      ""int C.<F>d__4.<>1__state""
-    IL_00f6:  ldarg.0
-    IL_00f7:  ldflda     ""System.Runtime.CompilerServices.AsyncTaskMethodBuilder C.<F>d__4.<>t__builder""
-    IL_00fc:  ldloc.s    V_4
-    IL_00fe:  call       ""void System.Runtime.CompilerServices.AsyncTaskMethodBuilder.SetException(System.Exception)""
-    IL_0103:  nop
-    IL_0104:  leave.s    IL_011a
-  }
-  IL_0106:  ldarg.0
-  IL_0107:  ldc.i4.s   -2
-  IL_0109:  stfld      ""int C.<F>d__4.<>1__state""
-  IL_010e:  ldarg.0
-  IL_010f:  ldflda     ""System.Runtime.CompilerServices.AsyncTaskMethodBuilder C.<F>d__4.<>t__builder""
-  IL_0114:  call       ""void System.Runtime.CompilerServices.AsyncTaskMethodBuilder.SetResult()""
-  IL_0119:  nop
-  IL_011a:  ret
-}
-");
+            diff2.VerifyIL("C.<F>d__4.System.Runtime.CompilerServices.IAsyncStateMachine.MoveNext", """
+            {
+              // Code size      285 (0x11d)
+              .maxstack  3
+              .locals init (int V_0,
+                            System.Runtime.CompilerServices.TaskAwaiter V_1,
+                            C.<F>d__4 V_2,
+                            System.Runtime.CompilerServices.TaskAwaiter V_3,
+                            System.Exception V_4)
+              IL_0000:  ldarg.0
+              IL_0001:  ldfld      "int C.<F>d__4.<>1__state"
+              IL_0006:  stloc.0
+              .try
+              {
+                IL_0007:  ldloc.0
+                IL_0008:  brfalse.s  IL_0012
+                IL_000a:  br.s       IL_000c
+                IL_000c:  ldloc.0
+                IL_000d:  ldc.i4.3
+                IL_000e:  beq.s      IL_0017
+                IL_0010:  br.s       IL_0019
+                IL_0012:  br         IL_00c2
+                IL_0017:  br.s       IL_0066
+                IL_0019:  ldloc.0
+                IL_001a:  ldc.i4.0
+                IL_001b:  blt.s      IL_002a
+                IL_001d:  ldstr      "Edit and Continue can't resume suspended asynchronous method since the corresponding await expression has been deleted"
```

Comment:
>"Edit and Continue can't resume suspended asynchronous method since the corresponding await expression has been deleted" [](http://example.com/codeflow?start=37&length=120)

Should this be a resource rather than a string?